test-configs.yaml

```diff
@@ -76,6 +76,13 @@ test_plans:
     filters:
       - blacklist: *kselftest_defconfig_filter
 
+  boot_fastboot:
+    name: 'boot-fastboot'
+    rootfs: debian_buster_ramdisk
+    pattern: 'boot-fastboot/generic-fastboot-boot-template.jinja2'
+    params:
+      job_timeout: '50'
```

Comment:
@khilman This template parameter sets the global job timeout, so it's not the default 10min in this case.

Reply:
@khilman Yes, the global job timeout job is 50 min being set by the parms job_timeout in the configs of the fastboot test plan.

Reply:
Can you guys confirm this is now resolved? @khilman @touilkhouloud